### PR TITLE
Add Inital draft of frontend changes & bootstrap additions

### PR DESCRIPTION
**Pull request by OnToNothing**
Initial draft of front-end changes. added additions will include updated modals for simple and advanced pool creation, selection for free tile, public vs private pools, image tags, and improved menu function

### CODE REVIEW

**Comment by jpappel** [comment]
looking good so far, I recommend taking a peak at #13 for the most up to date version of the api

File: src/app/templates/index.html [hunk 1]
```diff
@@ -1,75 +1,169 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <title>Bingo Card Maker</title>
-  <link rel="stylesheet" href="{{ url_for('static', filename='styles.css') }}">
+    <meta charset="UTF-8">
+    <title>Bingo Card Generator</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="src/app/static/styles.css">
```
Comment (drekdrek) [inline src/app/templates/index.html:7]: loading this css will 404. when using a template you should use 
`<link rel="stylesheet" href="{{ url_for('static', filename='styles.css') }}">` to load static files.

Comment (drekdrek) [reply src/app/templates/index.html:7]: this line is also duplicated at the end of the file

**Comment by jpappel** [comment]
Any progress/updates on this?

**Pull request closed without merging**